src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
You are moving  your return statement to a new line without use round brackets!

[Look here](https://mate-academy.github.io/style-guides/javascript.html#83-in-case-the-expression-spans-over-multiple-lines-wrap-it-in-parentheses-for-better-readability)